internal/powerplatform/provider/provider.go

```diff
@@ -517,3 +515,29 @@ func EnvDefaultFunc(k string, dv interface{}) string {
 	}
 	return ""
 }
+
+const (
+	// TestsUnitProviderConfig is a shared configuration to combine with the actual
+	// test configuration so the Power Platform client is properly configured.
+	// It is also possible to use the POWER_PLATFORM_ environment variables instead.
+	//lintignore:AT004
```

Comment:
See if we can fix this linter warning instead of suppressing

Reply:
to investigate in a separate work item